BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -305,7 +315,7 @@ - (NSURLSession *)backgroundSessionForId:(NSString *)backgroundSessionId
 
 - (void)pendingBackgroundDownloadUploadSessionTasks:(void (^)(NSArray<NSURLSessionUploadTask *> * uploadTasks, NSArray<NSURLSessionDownloadTask *> * downloadTasks))completion
 {
-    if (completion != nil) {
+    if (self.supportBackgroundSessionTasks == YES && completion != nil) {
```

Comment:
if there is a completion shouldn't there be an else where we return the 'not supported' error? That's what we've done elsewhere...